### PR TITLE
Feature/current jwt map for user

### PR DESCRIPTION
**Pull request by fsmicdev**

### CODE REVIEW

File: Apromore-Core-Components/Apromore-Portal/src/main/java/org/apromore/portal/ApromoreZKListener.java [hunk 1]
```diff
@@ -25,10 +25,7 @@
 import com.nimbusds.jwt.JWT;
 import com.nimbusds.jwt.JWTClaimsSet;
 import java.io.IOException;
-import java.util.Dictionary;
-import java.util.Enumeration;
-import java.util.HashMap;
-import java.util.Map;
+import java.util.*;
```
Comment (raboczi) [inline Apromore-Core-Components/Apromore-Portal/src/main/java/org/apromore/portal/ApromoreZKListener.java]: Avoid wildcarded imports -- it's actually one of our Checkstyle rules, so theoretically we'll have to put it all back in order to enable Checkstyle on the Portal.

File: Apromore-Core-Components/Apromore-Portal/src/main/java/org/apromore/portal/dialogController/UserMenuController.java [hunk 2]
```diff
@@ -28,11 +28,8 @@
 
 import java.io.UnsupportedEncodingException;
 import java.net.URLEncoder;
-import java.util.Comparator;
-import java.util.List;
-import java.util.Locale;
-import java.util.SortedMap;
-import java.util.TreeMap;
+import java.util.*;
```
Comment (raboczi) [inline Apromore-Core-Components/Apromore-Portal/src/main/java/org/apromore/portal/dialogController/UserMenuController.java:31]: Shouldn't use import wildcards.

**Review by raboczi — APPROVED** [review]
Style seems fine, builds but I don't have the KC setup to functionally test it locally.

I have a comment about avoiding import wildcards, but otherwise I approve this.

File: Apromore-Core-Components/Apromore-Portal/src/main/java/org/apromore/portal/ApromoreZKListener.java (hunk 1 above)
Comment (fsmicdev) [reply Apromore-Core-Components/Apromore-Portal/src/main/java/org/apromore/portal/ApromoreZKListener.java]: K, will fix now.

File: Apromore-Core-Components/Apromore-Portal/src/main/java/org/apromore/portal/dialogController/UserMenuController.java (hunk 2 above)
Comment (fsmicdev) [reply Apromore-Core-Components/Apromore-Portal/src/main/java/org/apromore/portal/dialogController/UserMenuController.java:31]: Cool - have removed.

**Pull request merged**